cf_xarray/units.py

```diff
@@ -83,13 +83,29 @@ def repl(m):
 )
 
 # Define commonly encountered units (both CF and non-CF) not defined by pint
+units.define("@alias meter = gpm")
+# ----- end block copied from metpy
+
+# -----
+# The following redefinitions were copied from xclim under the terms of their Apache-2 license
+# In pint, the default symbol for year is "a" which is not CF-compliant (stands for "are")
+units.define("year = 365.25 * day = yr")
+
+# Define commonly encountered units not defined by pint
+units.define("@alias degC = C = deg_C")
+units.define("@alias degK = deg_K")
+units.define("@alias day = d")
+units.define("@alias hour = h")  # Not the Planck constant...
 units.define(
-    "degrees_north = degree = degrees_N = degreesN = degree_north = degree_N = degreeN"
+    "degrees_north = 1 * degree = degrees_north = degrees_N = degreesN = degree_north = degree_N = degreeN"
```

Comment:
@aulemahal Why the `1 * `

Reply:
I'm not sure about the `1 *` anymore... It might be unnecessary, after testing it seems to serve no purpose...

However, the third element is the string used in the abbreviated version of the unit, that you get with `f"{unit:~}"` (which we use here in `short_formatter`). It's to ensure the correct version is printed, instead of `"degrees_N"` or `"degrees_E"`.